docs/libcurl/curl_easy_setopt.3

```diff
@@ -327,6 +327,12 @@ Disable Content decoding. See \fICURLOPT_HTTP_CONTENT_DECODING(3)\fP
 Disable Transfer decoding. See \fICURLOPT_HTTP_TRANSFER_DECODING(3)\fP
 .IP CURLOPT_EXPECT_100_TIMEOUT_MS
 100-continue timeout. See \fICURLOPT_EXPECT_100_TIMEOUT_MS(3)\fP
+.IP CURLOPT_HTTP_TRAILER_FUNCTION
```

Comment:
A little bikeshedding here perhaps, but...

I'm not convinced it is a good idea to include `HTTP` in the name. What if there's another protocol with a similar concept that we could use this callback for at a later point in time? Maybe just `CURLOPT_TRAILERFUNCTION`  ? Most of the existing *FUNCTION names are already named without a separating underscore.